pom.xml

```diff
@@ -424,7 +424,7 @@
     <dependency>
       <groupId>com.hms_networks.americas.sc</groupId>
       <artifactId>extensions</artifactId>
-      <version>1.12.1</version>
+      <version>1.13.9-pre1</version>
```

Comment:
Pre-release and other beta versions should not be used in an official connector release. PR is pending for the official 1.13.9 version already: https://github.com/hms-networks/sc-ewon-flexy-extensions-lib/pull/46

Reply:
@alexjhawk  I will put draft back on.

Reply:
@it-hms @oliver-walker-hms v1.13.9 of the extensions library was officially released earlier today. This commit can be updated with the proper version.

Reply:
this is updated